ci/setup_env.sh

```diff
@@ -63,29 +63,6 @@ conda update -n base conda
 echo "conda info -a"
 conda info -a
 
-echo
-echo "set the compiler cache to work"
-if [ -z "$NOCACHE" ] && [ "${TRAVIS_OS_NAME}" == "linux" ]; then
```

Comment:
umm is this just not used elsewhere? it seems the builds are now taking a lot longer.

Reply:
Thanks for the heads up. I double checked, and there is nowhere in our code where we set `TRAVIS_OS_NAME`, so I don't see how any of the conditions here can be true.

I checked the builds, and the only I see that was significantly slower was a Windows one, which doesn't use this file. I guess it's been just randomness, but in any case, I rebased, and if there is anything still significantly slow, I'll have another look.

Reply:
hmm yeah this might have been old ok great